### DISCUSSION TITLE
[2021.09.07] 입양시각구하기(2)

### DISCUSSION THREAD

**Discussion by hyk339** [pyunji/Hyundai-ITnE-Team_QuadCore#114] [SQL]
[입양시각구하기(2)](https://programmers.co.kr/learn/courses/30/lessons/59413)

**Comment by hyk339** [comment]
[입양시각구하기(2) solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/PR_59413.sql)

**Comment by chaeyeon4u** [comment]
[입양시각구하기(2) solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59413.sql)

**Comment by leechang4** [comment]
[입양시각구하기(2) solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR_59413.sql)

**Comment by pyunji** [comment]
[입양시각구하기(2) solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59413.sql)
